test/e2e/lib/components/site-editor-component.js

```diff
@@ -64,6 +65,25 @@ export default class SiteEditorComponent extends AsyncBaseContainer {
 		);
 	}
 
+	async isBlockInserterOpen() {
+		const inserterMenuLocator = By.css( '.block-editor-inserter__menu' );
+		return await driverHelper.isElementLocated( this.driver, inserterMenuLocator );
+	}
+
+	async openBlockInserter() {
+		const inserterToggleLocator = By.css(
+			'.edit-site-header .edit-site-header-toolbar__inserter-toggle'
+		);
+		await driverHelper.clickWhenClickable( this.driver, inserterToggleLocator );
+		// "Click" twice - the first click seems to trigger a tooltip, the second opens the menu
+		// See https://github.com/Automattic/wp-calypso/issues/43179
+		if ( ! ( await this.isBlockInserterOpen() ) ) {
+			await driverHelper.clickWhenClickable( this.driver, inserterToggleLocator );
+		}
```

Comment:
This part doesn't seem to be needed? When running locally, it opens the Inserter for me consistently with single click only 👍 
```suggestion
```

Reply:
Tried to remove it triggers the tooltip for me, rather than opening the menu.